### ISSUE TITLE
Machines list not updated in webUI (Nginx) after VM change

### ISSUE THREAD

**Issue by xECK29x** [cuckoosandbox/cuckoo#2185]
Thanks for creating an issue! But first: did you read our community guidelines?
https://cuckoo.sh/docs/introduction/community.html

**Comment by doomedraven** [comment]
you need to restart cuckoo for that

**Comment by xECK29x** [comment]
I have, along with nginx for good measure as well as clearing my browser cache.

**Comment by doomedraven** [comment]
weird, if you start standalone webgui you can see change? just to discart something on cuckoo side, and be more sure what that is nginx problem, is there also uwsgi between cuckoo and nginx?

**Comment by xECK29x** [comment]
Yes uwsgi is used, and yes this is isolated to nginx, works fine with standard cuckoo web server, so I assume it could be a number of things causing the disconnect. Did not see anything glaring in the access.log.

**Comment by vladbrinzea** [comment]
Had a similar issue, was seeing "unintentionally cached" values when I was just restarting nginx post config-changes. 
Fixed it by manually stopping nginx and afterwards uwsgi via systemctl and then restarting them (uwsgi first, then nginx). All of this was done with cuckoo stopped, when I brought it back up all the "unintentionally cached" values were cleaned up.

Short answer I guess is you need to restart both uwsgi and nginx to force a reload, not just nginx since it's uwsgi that holds the actual website content not nginx.
